tests/appsec/iast_packages/packages/pkg_cachetools.py

```diff
@@ -0,0 +1,46 @@
+"""
```

Comment:
Were these files generated somehow? Documentation might be a good idea in that case.

Reply:
Semi-automatically generated with IA help, there is a template in tests/appsec/iast_packages/packages/pkg_template.py.tpl